src/libs/Navigation/AppNavigator/getRootNavigatorScreenOptions.js

```diff
@@ -0,0 +1,65 @@
+import modalCardStyleInterpolator from './modalCardStyleInterpolator';
+import styles from '../../../styles/styles';
+import variables from '../../../styles/variables';
+
+const commonScreenOptions = {
+    headerShown: false,
+    gestureDirection: 'horizontal',
+    animationEnabled: true,
+    cardOverlayEnabled: true,
+    animationTypeForReplace: 'push',
+};
+
+export default (isSmallScreenWidth) => ({
+    rightModalNavigator: {
+        ...commonScreenOptions,
+        cardStyleInterpolator: (props) => modalCardStyleInterpolator(isSmallScreenWidth, false, props),
+        presentation: 'transparentModal',
+
+        // We want pop in RHP since there are some flows that would work weird otherwise
+        animationTypeForReplace: 'pop',
+        cardStyle: {
+            ...styles.cardStyleNavigator,
+
+            // This is necessary to cover translated sidebar with overlay.
+            marginLeft: isSmallScreenWidth ? 0 : -2 * variables.sideBarWidth,
+        },
+    },
+
+    homeScreen: {
+        title: 'New Expensify',
+        ...commonScreenOptions,
+        cardStyleInterpolator: (props) => modalCardStyleInterpolator(isSmallScreenWidth, false, props),
+
+        // Prevent unnecessary scrolling
```

Comment:
I know this existed before you added the change, but do you or @mountiny know, what do we mean here by `Prevent unnecessary scrolling`?